dist/P2P/MulticastPeerDiscovery.js

```diff
@@ -0,0 +1,175 @@
+// import PeerStore from '../PeerStore/PeerStore'
```

Comment:
This is all commented out? It should be an in issue if it's not yet ready?

Reply:
I have just forgotten to build into dist before I committed.